requirements.txt

```diff
@@ -32,4 +32,5 @@ netifaces==0.10.4
 pyjokes==0.5.0
 psutil==4.1.0
 pep8==1.7.0
+python-nest==2.9.2
```

Comment:
I didn't see this library been used on the code. Is that the binary `nest` from the subprocess call ?